.github/workflows/ci.yml

```diff
@@ -13,29 +13,58 @@ jobs:
     steps:
       - name: Clone This Repo
         uses: actions/checkout@v2
-      - name: Build
+      - name: Build with Sanitizers
         run: |
-          sudo apt-get install -y lcov
-          cmake -S test -B build/ \
+          sudo apt-get install -y cmake lcov
+          CFLAGS="  --coverage -O0 -Wall -Wextra"
+          CFLAGS+=" -Werror -Wno-error=pedantic"
```

Comment:
Is `-Wno-error=pedantic` this suppressing the treatment of `-Wpedantic` warning as an error? If yes, what's the reason behind it?

Reply:
Yes. This disables -Werror for -Wpedantic so that we get warnings rather than errors. This is due to the use of ending commas in enum definitions in the 3rd-party http-parser library.